.travis.yml

```diff
@@ -24,6 +34,20 @@ before_install:
 - wget http://get.sensiolabs.org/php-cs-fixer.phar -O php-cs-fixer
 - sudo mv php-cs-fixer /usr/local/bin
 - sudo chmod +x /usr/local/bin/php-cs-fixer
+# We want to fail fast for coding standard violations.
+- php-cs-fixer fix --dry-run --diff --level=psr2 --fixers=concat_with_spaces .
+- if [ ! -d ${HOME}/gcloud/google-cloud-sdk ]; then
+      mkdir -p ${HOME}/gcloud &&
+      wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=${HOME}/gcloud &&
+      cd ${HOME}/gcloud &&
+      tar xzf google-cloud-sdk.tar.gz &&
+      printf '\ny\n\ny\ny\n' | ./google-cloud-sdk/install.sh &&
```

Comment:
Also there is an environment variable `CLOUDSDK_CORE_DISABLE_PROMPTS` and added it.